src/assets/scripts/client/InputController.js

```diff
@@ -975,21 +975,36 @@ export default class InputController {
         return this._aircraftCommander.runCommands(aircraft, parsedCommand.args);
     }
 
+    /**
+     * Retrieve and return the x/y offset from the airport center (km) where the user clicked in the provided event
```

Comment:
I know what you're saying here, but the wording is a bit tricky.  Maybe something like:

```txt
Given a user mouse click, retrieve and return x/y offset from airport center
```

Reply:
`Given a mouse click event, retrieve and return the [x, y] offset from the airport center, in km`